.gitignore

```diff
@@ -51,6 +51,7 @@ local.properties
 *.iml
 /android/
 /ReactAndroid/packages/
+.settings/
```

Comment:
Android Studio leaves these `.settings/` folders sprinkled around the repo and I was sick of it.